packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -304,22 +310,43 @@ contract Deploy is Deployer {
         }
         di.run(dii, dio);
 
-        save("L1CrossDomainMessengerImpl", address(dio.l1CrossDomainMessengerImpl()));
-        save("OptimismMintableERC20FactoryImpl", address(dio.optimismMintableERC20FactoryImpl()));
-        save("SystemConfigImpl", address(dio.systemConfigImpl()));
-        save("L1StandardBridgeImpl", address(dio.l1StandardBridgeImpl()));
-        save("L1ERC721BridgeImpl", address(dio.l1ERC721BridgeImpl()));
+        save(string.concat("L1CrossDomainMessengerImpl", _suffix), address(dio.l1CrossDomainMessengerImpl()));
+        save(
+            string.concat("OptimismMintableERC20FactoryImpl", _suffix), address(dio.optimismMintableERC20FactoryImpl())
+        );
+        save(string.concat("SystemConfigImpl", _suffix), address(dio.systemConfigImpl()));
+        save(string.concat("L1StandardBridgeImpl", _suffix), address(dio.l1StandardBridgeImpl()));
+        save(string.concat("L1ERC721BridgeImpl", _suffix), address(dio.l1ERC721BridgeImpl()));
 
         // Fault proofs
-        save("OptimismPortalImpl", address(dio.optimismPortalImpl()));
-        save("OptimismPortal2Impl", address(dio.optimismPortalImpl()));
-        save("DisputeGameFactoryImpl", address(dio.disputeGameFactoryImpl()));
-        save("DelayedWETHImpl", address(dio.delayedWETHImpl()));
-        save("PreimageOracleSingleton", address(dio.preimageOracleSingleton()));
-        save("MipsSingleton", address(dio.mipsSingleton()));
-        save("OPContractsManager", address(dio.opcm()));
-
-        Types.ContractSet memory contracts = _impls();
+        save(string.concat("OptimismPortalImpl", _suffix), address(dio.optimismPortalImpl()));
+        save(string.concat("OptimismPortal2Impl", _suffix), address(dio.optimismPortalImpl()));
+        save(string.concat("DisputeGameFactoryImpl", _suffix), address(dio.disputeGameFactoryImpl()));
+        save(string.concat("DelayedWETHImpl", _suffix), address(dio.delayedWETHImpl()));
+        save(string.concat("PreimageOracleSingleton", _suffix), address(dio.preimageOracleSingleton()));
+        save(string.concat("MipsSingleton", _suffix), address(dio.mipsSingleton()));
+        save(string.concat("OPContractsManager", _suffix), address(dio.opcm()));
+
+        // Get a contract set from the implementation addresses with the suffix.
+        // Get a contract set from the implementation addresses just deployed
```

Comment:
nit, merge these comments into one sentence? this reads a bit confusingly

Reply:
16d54d6